src/SFA.DAS.ProviderFunding.Web/Services/CSVBuilder.cs

```diff
@@ -0,0 +1,41 @@
+﻿using System.Collections.Generic;
+using System.Data;
+using System.Linq;
+using SFA.DAS.ProviderFunding.Web.Models;
+
+namespace SFA.DAS.ProviderFunding.Web.Services
+{
+    public class CSVBuilder
```

Comment:
It's not actually building a CSV in this class is it?

Reply:
no? and this is still the case now? should we update the name?